src/main/java/sawkiewicz/task1/Item.java

```diff
@@ -0,0 +1,34 @@
+package sawkiewicz.task1;
+
+public class Item {
+    private String name;
+    private int quantity;
```

Comment:
should not have `qunatity` because you need to support such flow

```
1) client A starts checkout
2) client A adds item 1 to he's basket
3) client B starts checkout
4) client B adds item 1 to he's basket

```

so you need to have a representation of `Order` in your code